testing/tests/DevExpress.ui.widgets/ganttParts/events.tests.js

```diff
@@ -116,14 +116,14 @@ QUnit.module('Events', moduleConfig, () => {
         this.createInstance(my_options);
         this.clock.tick();
 
-        const $top_item = $('body').find('.gst_item');
+        const $top_item = $('#qunit-fixture').find('.gst_item');
```

Comment:
Why is it necessary?

Reply:
because shadowDom does not have the `body` selector